.studio/infra-proxy-service

```diff
@@ -24,6 +24,18 @@ document "infra_service_load_sample_data" <<DOC
   infra_service_load_sample_data -N 100 -S chef-server -O chef-org
 DOC
 
+function start_local_dev_chef_server() {
```

Comment:
this is a duplicate of `start_chef_server` from `.studio/chef-server-collection`, do we need this?

Reply:
Yes, it is duplicate of `start_chef_server` but here, it gets added with guard condition here to avoid config patches every time.

Oh yeah, let's wrap `start_chef_server` with if the condition itself. Thanks